iohk-monitoring/src/Cardano/BM/Backend/Switchboard.lhs

```diff
@@ -25,8 +25,8 @@ module Cardano.BM.Backend.Switchboard
     , addExternalBackend
     , addExternalScribe
     -- * testing
-    , realizeSwitchboard
-    , unrealizeSwitchboard
+    --, realizeSwitchboard
+    --, unrealizeSwitchboard
```

Comment:
If these are commented out - why not just remove them?

Reply:
for the time being we want to keep them for testing. later, they can be removed.